### PR TITLE
expanding tests coverage

### PR DESCRIPTION
**Pull request by amateja**
Reviewing recent issues and preparing recent PR I've written some additional tests which expands tests coverage. Hope you'll find them useful.

### CODE REVIEW

File: tests/test_widgets.py [hunk 1]
```diff
@@ -8,22 +9,33 @@
 )
 
 
+class PhonePrefixSelectTest(SimpleTestCase):
+    def test_without_babel(self):
+        import phonenumber_field
+
+        babel_module = phonenumber_field.widgets.babel
+        phonenumber_field.widgets.babel = None
```
Comment (francoisfreitag) [inline tests/test_widgets.py:17]: Please make sure the cleanup is always executed, e.g. with https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup.

**Review by francoisfreitag — APPROVED** [review]
Otherwise LGTM, thanks!

File: tests/test_widgets.py (hunk 1 above)
Comment (amateja) [reply tests/test_widgets.py]: Good point. I think using `setUp` and `tearDown` should be enough. The only difference between `tearDown` and registering a function with `addCleanup` is that if won't be fired if `setUp` fails whilst cleanup method is fired always.

**Comment by francoisfreitag** [comment]
Thanks for the update. Pushed cosmetic edits. If they look good to you, I’ll merge this.

**Comment by amateja** [comment]
I totally don't mind rephrasing commit messages or splitting and joining my commits. Thanks for your help.

**Pull request merged**